news/2019-07-07-beatmap-spotlights-spring-2019.md

```diff
@@ -0,0 +1,327 @@
+
+A mapper who is reputable in the community for their unorthodox music taste and style, [squirrelpascals](https://osu.ppy.sh/users/6151332), brings forth a very fitting interpretation of Hyakkei's "kagefumi". 
+
+This mapping contains unique stream patterns, minimalistic slider velocities, characterized slider shapes, and much more. squirrelpascals underlines the core portions of this chill, tranquil, math rock song with clarity and accentuates their features fantastically through distinctive design choices that align with the song's simplicity.
+
+Those unprepared for the challenge may find a very engaging experience in the insane difficulty. For the more skilled players, the top difficulty is not something you would want to sleep on either. With more emphasis on greater slider movements, the difficulties compliment each other with their slight varied approaches to the song; while both capturing it's integrity faithfully. This is a truly refreshing mapset overall that stands out among the many maps in the ranked section!
+
+_written by [faygo](https://osu.ppy.sh/users/5199686)_
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/osu/uh.jpg)](https://osu.ppy.sh/beatmapsets/948199)
+
+[sammish](https://osu.ppy.sh/users/4415076) manages to make 17 minutes feel like 5 in his map of Tennyson's "Uh Oh! EP". 
+
+The hard work he puts into expressing each song's personality shows — and obviously pays off. The map feels smooth to play all the way through and aligns very closely to Tennyson's unique, funky style of music. 
+
+This is a phenomenal debut from both sammish and Tennyson in the ranked section and we hope to see more from them in the future!
+
+_written by [squirrelpascals](https://osu.ppy.sh/users/6151332)_
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/osu/meng.jpg)](https://osu.ppy.sh/beatmapsets/927682)
+
+[AIR](https://osu.ppy.sh/users/2070688) displays their usual prowess in Yin Lin's "Lu Gu Xun Meng", a traditional-sounding Chinese ballad. They skillfully craft dynamic jump patterns through expert use of slider velocity changes, repeating slider shapes, and varied geometric arrangements. 
+
+As always, the map has a lot of contrast, with every separate section of the map feeling a unique and close match to the features of the song. The hitsounding, also done by AIR, follows the same idea, adding to the experience. If the top difficulty is a bit too jumpy, be sure to check out the Insane and Hard which maintain the same concepts as the top difficulty, but at a more manageable level.
+
+_written by [Nao Tomori](https://osu.ppy.sh/users/5364763)_
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/osu/kyoumen.jpg)](https://osu.ppy.sh/beatmapsets/731108)
+
+Like last time, we present you another set of "Kyoumen no Nami" by YURiKA. This time, it is [Vell](https://osu.ppy.sh/users/3414541)'s turn to show what amazing things can be done with this breathtaking song. 
```

Comment:
Last time we showed you the orchestral arrangement of the song "Kyoumen no Nami" by YURiKA, but this time we have the stunning original full version. his time it's [Vell](https://osu.ppy.sh/users/3414541)'s turn to show what amazing things you can do with this breathtaking song.